### DISCUSSION TITLE
一个令我好奇的问题

### DISCUSSION THREAD

**Discussion by KongBai000-0** [share121/inter-knot#4987] [其他]
![QQ图片20240818000820](https://github.com/user-attachments/assets/c3348c55-715a-464a-8d85-c9871252dd5e)
突然想到一个问题：为什么米家游戏现在玩了也被骂特别是原神其他游戏就算是同类型的也不见得哪个这么猛社区环境差的又不是只有米为什么骂这个骂的最起劲？

**Comment by passbyDFX** [comment]
因为骂了有流量，可以赚钱，一些路人不知道的随波逐流一起骂了

**Comment by Shirin-k423** [comment]
因为骂米有流量，有流量就能赚钱。一切都是有一条黑色产业链的，一些不坏好意的公司发布悬赏，下边外包接受悬赏再在网络招募水军，按一条黑评多少钱来抹黑，然后去带歪路人，让路人把生活中憋的气全宣泄到原神上去。又刚好现如今网络属于后真相时代，真相根本就不重要，简单点说我抹黑原神抄袭了塞尔达，我需要理由吗？需要证据吗？不需要，但你澄清没有抄袭反而需要理由证据，并且这些没人在乎。小部分人为了利益大部分人为了宣泄情绪，而原神恰好站在了他们获得利益和宣泄情绪的一道口子

**Comment by ParanoidHikari** [comment]
因为这已经变成一种网络zzzq了，而且这个被害人还是一团棉花随你打骂，短时间内你没法在互联网再找到一个这样逆来顺受、打不还手骂不还口、造谣抹黑百无禁忌的对象了。米的存在大到无法忽视却又无法在社区自卫，招来的恶意只会增长而不会消失。寄生在米身上的蛀虫心安理得地啃噬米的血肉，而无需付出任何代价。

**Comment by voidomaino** [comment]
米哈游兴起这几年是国内舆论和经济环境剧变的几年。

他们幸运在于所有人退步的时候进步，不幸在于本该所有人退步的时候进步。

**Comment by TNCNET** [comment]
side A
当然也有一些人本身利益和这个相关；比如一些水军和“50w”，以及好战分子；有些时候，如果你仔细观察，这些人都可能是同一类人，所以难办。［这种情况是真实存在的］

毕竟米忽悠的崛起，代表着国内有相当不错的作品出海了，有影响力了，也许这让那些“唱衰”国内情况的人肯定不满。

至于水军？他们拿钱办事的。之后完成任务就跑路。

Side B
还有些人就是跟风，或者确实不是上面（包括楼上几位）说的原因的。包括其他同行；

还有一些主机玩家，这类人的意见是，「你一个移动端游戏怎么敢和我们已有比，甚至比我们已有的好？」

另一种情况是，因为原的圈子本身就是人多，而自然，人多的地方就是总有混乱之处的。那么也确实有一些“brain rot”的玩家到处动不动就刷相关内容，而这种行为肯定是被反感的。

side c
除了上面这种“brain rot”玩家外，就是楼上几位说的情况了。还有些人，本身就是，你选A，我就选B或者选C。故意这样的。

side d
一些内鬼传递的不实或者企业机密信息，加上内鬼们的添油加醋，那么很多事情在之后的发展中，实际上就没有发生，或者发展并不是他们所说。而跟风者和brain rot人是很容易把这些当回事，甚至当真。

毕竟也有半真半假的情况，一半是确实发生了的，另一半就不是一回事。

**Comment by Fluorine-z** [comment]
甚至有段时间有些专门骂原神的up投视频都是原神创作者激励，相当于拿原的钱骂原，不是非常理解原那边也不管社区是什么原因

**Comment by 11-billy** [comment]
在国内有硬核联盟这一类手游吸血鬼，大公司还好，小的工作室基本上就是白打工了，米发行的游戏基本都绕过它们了，你赚钱了还不交保护费，能不搞你吗？
